src/components/interaction-designer/Block.vue

```diff
@@ -64,6 +66,16 @@
               class="fa-btn"
               @click.prevent="handleDuplicateBlock" />
           </div>
+          <!--Expand block editor-->
+          <div class="mr-1 ml-2">
+            <i
+              v-if="isEditable"
+              class="glyphicon"
+              :class="{'glyphicon-resize-small': showBlockEditor, 'glyphicon-resize-full': !showBlockEditor}"
```

Comment:
@poojahpatil90 , I think you're right (from Slack), we should use custom icon here.
To make this easy, what if you:
- merge my branch https://github.com/FLOIP/flow-builder/pull/117 into yours
- edit the target branch of this current PR
- pick the custom icon I made there for `minimize` & `expand`

Are you ok with that ?

Reply:
Done